bot/code_review_bot/workflow.py

```diff
@@ -322,6 +326,12 @@ def _in_group(dep_id):
                     continue
                 artifacts = task.load_artifacts(self.queue_service)
                 if artifacts is not None:
+                    if 'public/firefox-source-docs-url.txt' in artifacts:
+                        link_to_doc = artifacts['public/firefox-source-docs-url.txt'].decode('UTF-8')
+                        self.phabricator.comment(
```

Comment:
Maybe it should be part of the summary comment rather than a separate comment, wdyt @sylvestre?